source/code/plugins/security_lib.rb

```diff
@@ -13,6 +13,12 @@ def self.log_type_mapping
       @@LOG_TYPE_MAPPING
     end
 
+    def self.get_ident(ident)
+      return 'CEF' if ident.include?('CEF')
+      return '%ASA' if ident.include?('%ASA')
+      return nil
```

Comment:
Can you add a trace here in case there's not match. Ideally a debug trace that isn't written by default